### ISSUE TITLE
Installation via package manager fails

### ISSUE THREAD

**Issue by davisjam** [CamFlow/documentation#1]
# Problem

I am trying to install CamFlow. I have a fresh Fedora install and am trying to follow the "[install via the package manager](https://github.com/CamFlow/documentation/blob/master/docs/installation.md#1-installation-via-the-package-manager)" steps.

The first step to update my repositories succeeded.
When I ran `sudo dnf install camflow`, I get

```
No match for argument: camflow
Error: Unable to find a match
```

I then ran `yum repolist all` and was prompted several times for GPG keys for various packagecloud.io repositories. After accepting these I see repo ids "camflow_provenance" and "camflow_provenance-source", but the `sudo dnf install camflow` command still fails as before.

# Environment

I am using Virtualbox.
My VM is running a fresh install of Fedora 28 Workstation edition.

**Comment by tfjmp** [comment]
The problem is fedora 28. As you can see [here](https://packagecloud.io/camflow/provenance), the distributed packages are for Fedora 27. It is likely that next release will be for fedora 28.

I will state in the instructions should check which version of Fedora the packages are for.

**Comment by davisjam** [comment]
Fixed in 62da352030d480966fa4ff206341dfed0d7255b4.
